### PR TITLE
16222, 16384 - Affiliation Invitation - Access Request - Authorization & Emails

### PR DESCRIPTION
**Pull request by hfekete**
https://github.com/bcgov/entity/issues/16222
https://github.com/bcgov/entity/issues/16384

Add email templates and email invocation on the service side
Finalize authorization actions for affiliation invitations

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Note:** This request has been sent to all Account Administrators and Coordinators of {{ to_acount_name }}.
+
+To manage this request from **{{ from_account_name }}** to manage Registries activities for {{ to_acount_name }} follow the link below:
+
+To authorise or not authorise **{{ from_account_name }}** to manage Registries activities for **{{ business_name }}**, log in to your BC Registries Account:
+
+
+If you have questions about this request please contact the Business Registry:
+[Business Registry contact component]
```
Comment (hfekete) [inline queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html:32]: Can someone suggest how to add this ?
I tried to figure it out, and am not sure why, when sending email to my Gmail account I dont see these, only Header I think.

File: queue_services/account-mailer/src/account_mailer/worker.py [hunk 2]
```diff
@@ -203,6 +203,23 @@ async def process_event(event_message: dict, flask_app):
                 template_name=template_name,
                 subject=subject, **email_msg)
 
+        elif message_type == MessageType.AFFILIATION_INVITATION_REQUEST.value \
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/worker.py:206]: minor suggestion:
```
message_type in [MessageType.AFFILIATION_INVITATION_REQUEST.value, 
MessageType.AFFILIATION_INVITATION_REQUEST_AUTHORIZATION.value]
```

File: queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Filing of Annual Reports, and
+* Changing the records office of the business.
+
+{% if additional_message %}
+**Message from the Requesting Account:**
+
+{{ additional_message }}
+{% endif %}
+
+**Note:** This request has been sent to all Account Administrators and Coordinators of {{ to_acount_name }}.
```
Comment (seeker25) [inline queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html:24]: Typo `to_account_name`

File: queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html (hunk 1 above)
Comment (seeker25) [reply queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html:32]: This looks like something from footer you need?

Comment (seeker25) [reply queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html:32]: image

**Review by seeker25 — APPROVED** [review]
Looks good to me, just a few minor suggestions - Looks like the linter wanted to use `in` as well, plus some import linting issues. 

Something like these commands should help you figure out the order of the imports:

`flake8 src/auth_api tests --isort-show-traceback`

` . venv/bin/activate && flake8 src/account_mailer tests --isort-show-traceback`

Please make sure the CI passes before merging.

File: queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html (hunk 1 above)
Comment (seeker25) [reply queue_services/account-mailer/src/account_mailer/email_templates/affiliation_invitation_request.html:32]: Just put it right in the template like this one:

image

**Pull request merged**